src/deepstream.io.js

```diff
@@ -255,18 +257,11 @@ Deepstream.prototype._showStartLogo = function() {
 		return;
 	}
 	/* istanbul ignore next */
-	var logo =
-	' _____________________________________________________________________________\n'+
-	'                                                                              \n'+
-	'         /                                                             ,      \n'+
-	' ----__-/----__----__------__---__--_/_---)__----__----__---_--_-----------__-\n'+
-	'   /   /   /___) /___)   /   ) (_ ` /    /   ) /___) /   ) / /  )    /   /   )\n'+
-	' _(___/___(___ _(___ ___/___/_(__)_(_ __/_____(___ _(___(_/_/__/__o_/___(___/_\n'+
-	'                       /                                                      \n'+
-	'                      /                                                       \n'+
-	'=============================== STARTING... ==================================\n';
+	var logo = fs.readFileSync( './ascii-logo.txt', 'utf8' );
+
 	/* istanbul ignore next */
-	process.stdout.write( this._options.colors ? logo.yellow : logo );
+	process.stdout.write( logo + EOL );
+	process.stdout.write( ' =========================   ' + ( this._options.colors ? 'starting'.cyan : 'starting' ) + '   ==========================' + EOL );
 };
```

Comment:
Yes, it applies only to the logo.
Definition of the color value is done at 2 placed in the code currently
1: https://github.com/deepstreamIO/deepstream.io/blob/d9e751d26b4e067e57c09ebbfa38cb12b944eeb5/src/default-options.js#L10
2: https://github.com/deepstreamIO/deepstream.io/blob/a734d505a35cc6f9bc45e6ff2990a041b971f3c4/bin/deepstream-start.js#L30

We could introduce a public API to set the color and set it [here like the level](https://github.com/deepstreamIO/deepstream.io/blob/b24a0f6a9b3c142a74ce4c0603eaa5ae27ca68ba/src/config/config-initialiser.js#L93-L97) but this makes only sense for a console type/transport, so not sure if it's really worth.